src/main/java/ca/vapurrmaid/discretemathapplications/controllers/PrimeController.java

```diff
@@ -53,4 +54,28 @@ public ResponseEntity primeFactorization(@RequestParam("number") Integer number)
         }
     }
 
+    @GetMapping(
+            value = "/gcf",
+            params = {"n1", "n2", "n"},
+            produces = "application/json"
+    )
+    public ResponseEntity GCF(@RequestParam("n1") Integer num1, @RequestParam("n2") Integer num2, @RequestParam("n") List<Integer> others) {
+        try {
+            if (others.size() > 8) {
+                throw new Exception();
```

Comment:
Move new Issue/Design - Create meaningful controller validations and exceptions etc